### DISCUSSION TITLE
Expr misses some ColumnTrait operators, e.g. `starts_with`

### DISCUSSION THREAD

**Discussion by boan-anbo** [SeaQL/sea-orm#1944] [Q&A]
I was converting `ColumnTrait` conditions to `Expr` and found that Expr does not have some operator methods that ColumnTrait has such as [`starts_with`](https://docs.rs/sea-orm/0.12.3/sea_orm/entity/trait.ColumnTrait.html#method.starts_with) and `ends_with`. 

I'm not sure if this was intentional. 

My use case needs `Expr` to disambiguate column names on joined queries. 

Please kindly let me know if there is any workaround for me. Thanks!

**Comment by tyt2y3** [comment]
starts_with or ends_with is a higher-level convenience method over Expr, so this is by design.

> My use case needs Expr to disambiguate column names on joined queries.

Can you elaborate on your needs?
